test/db.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 var (
-	url      = "localhost:27017"
+	url      = "localhost:27017/?replicaSet=rs0&directConnection=true"
```

Comment:
Please update the readme with instructions on test setup (how to run the docker container and any other prereqs)

Reply:
Done in ad0f809783b0275dfc5f062ba4c81d4bfe69609a.